doc/user-guide/push-notifications/Push-notifications-client-side.md

```diff
@@ -0,0 +1,162 @@
+## Using push notifications on the client side
+
+There are just a few things the XMPP client application needs to receive the push notifications.
+One picture is worth a thousand words, hence, here you have two diagrams to follow:
+
+#### PubSub-full:
+
+ ![](pubsub_full_push_notifications_example_message_flow.png)
+
+#### PubSub-less:
+
+ ![](pubsub_less_push_notifications_example_message_flow.png)
```

Comment:
This diagram, even though the source image is of the same size as the one above, is rendered as a much bigger. This makes an impression (at least for me) that the PubSub-less configuration is more difficult then PubSub-full.